services/ros/ros-version.service.js

```diff
@@ -43,12 +43,12 @@ const packageSchema = Joi.object({
 export default class RosVersion extends GithubAuthV4Service {
   static category = 'version'
 
-  static route = { base: 'ros/v', pattern: ':distro/:packageName' }
+  static route = { base: 'ros/v', pattern: ':distro/:repoName' }
 
   static examples = [
     {
       title: 'ROS Package Index',
```

Comment:
Given the confusion so far, as well as changing the internal terminology (var/test names, etc) shall we attempt to clarify this a bit more in the user-facing text? We could consider changing the `title` here to clarify that the 'unit of versioning' (for want of a better term) here is the repo not the package. The other thing we sometimes do is include a documentation key in the example (e.g: https://github.com/badges/shields/blob/23c0406bedfc6930735e8f5ea75dfe34faf1f290/services/osslifecycle/osslifecycle.service.js#L3-L12 ) with some extra info that is shown to the user in the builder modal. We could use this to explain what the `repoName` param corresponds to in ROS-land or link to relevant documentation. I'm not 100% sure what to suggest as the platform specifics of ROS are out of my wheelhouse, but perhaps we can do a bit more to explain.

Reply:
Added some documentation:

<img width="852" alt="image" src="https://user-images.githubusercontent.com/14237/183734424-2ce87159-40f0-4718-a569-0cce477096bb.png">